nodejs/eks/cni.ts

```diff
@@ -87,21 +117,24 @@ function computeVpcCniYaml(yamlPath: string, args: VpcCniInputs): string {
     // Rewrite the envvars for the CNI daemon set as per the inputs.
     const daemonSet = cniYaml.filter(o => o.kind === "DaemonSet")[0];
     const env = daemonSet.spec.template.spec.containers[0].env;
-    if (args.nodePortSupport !== undefined) {
+    if (args.nodePortSupport) {
         env.push({name: "AWS_VPC_CNI_NODE_PORT_SUPPORT", value: args.nodePortSupport ? "true" : "false"});
     }
-    if (args.customNetworkConfig !== undefined) {
+    if (args.customNetworkConfig) {
         env.push({name: "AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG", value: args.customNetworkConfig ? "true" : "false"});
     }
-    if (args.externalSnat !== undefined) {
+    if (args.externalSnat) {
         env.push({name: "AWS_VPC_K8S_CNI_EXTERNALSNAT", value: args.externalSnat ? "true" : "false"});
     }
-    if (args.warmEniTarget !== undefined) {
+    if (args.warmEniTarget) {
         env.push({name: "WARM_ENI_TARGET", value: args.warmEniTarget.toString()});
     }
-    if (args.warmIpTarget !== undefined) {
+    if (args.warmIpTarget) {
         env.push({name: "WARM_IP_TARGET", value: args.warmIpTarget.toString()});
     }
+    env.push({name: "AWS_VPC_K8S_CNI_LOGLEVEL", value: args.logLevel ? args.logLevel.toString() : "DEBUG"});
+    env.push({name: "AWS_VPC_K8S_CNI_LOG_FILE", value: args.logFile ? args.logFile.toString() : "stdout"});
```

Comment:
Are these not the existing defaults?  I expected these to be similar to the above (using `if`) but perhaps this is intentional?

Reply:
Yes, `DEBUG` is a default that was being set in the YAML manifest, but I moved it into `cni.ts` and removed it from the `YAML` manifest to make it user-configurable.

I did not originally use `if` statements, as this would then only set `logLevel` and `logFile` if the user provided them.

But if we adopt the override approach then we'd address your feedback.